src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisShardSyncer.java

```diff
@@ -103,11 +106,16 @@ public synchronized void checkAndCreateLeasesForNewShards(IKinesisProxy kinesisP
     // CHECKSTYLE:OFF CyclomaticComplexity
     private synchronized void syncShardLeases(IKinesisProxy kinesisProxy,
```

Comment:
Let's overload this method to have a non shardlist param

Reply:
Addressed